pkg/bootstrap/option/instances.go

```diff
@@ -31,13 +31,14 @@ type (
 )
 
 const (
-	LocalhostIPv4       LocalhostValue       = "127.0.0.1"
-	LocalhostIPv6       LocalhostValue       = "::1"
-	WildcardIPv4        WildcardValue        = "0.0.0.0"
-	WildcardIPv6        WildcardValue        = "::"
-	DNSLookupFamilyIPv4 DNSLookupFamilyValue = "V4_ONLY"
-	DNSLookupFamilyIPv6 DNSLookupFamilyValue = "V6_ONLY"
-	DNSLookupFamilyIPS  DNSLookupFamilyValue = "ALL"
+	LocalhostIPv4           LocalhostValue       = "127.0.0.1"
+	LocalhostIPv6           LocalhostValue       = "::1"
+	WildcardIPv4            WildcardValue        = "0.0.0.0"
+	WildcardIPv6            WildcardValue        = "::"
+	DNSLookupFamilyIPv4     DNSLookupFamilyValue = "V4_ONLY"
+	DNSLookupFamilyIPv4Pred DNSLookupFamilyValue = "V4_PREFERRED"
+	DNSLookupFamilyIPv6     DNSLookupFamilyValue = "V6_ONLY"
+	DNSLookupFamilyIPS      DNSLookupFamilyValue = "ALL"
```

Comment:
nit: name these the same as envoy (DNSLookupFamily_V4_PREFERRED)

Reply:
Hi @howardjohn, do you mean it should be :

```
DNSLookupFamilyIPv4     DNSLookupFamilyValue = "V4_ONLY"
	DNSLookupFamilyIPv4Pred DNSLookupFamilyValue = "Cluster_V4_PREFERRED"
	DNSLookupFamilyIPv6     DNSLookupFamilyValue = "Cluster_V6_ONLY"
	DNSLookupFamilyIPS      DNSLookupFamilyValue = "Cluster_ALL"
```
am I right?